packages/mobile/locales/es-419/backupKeyFlow6.json

```diff
@@ -31,40 +31,40 @@
   },
   "socialBackupIntro": {
     "header": "Cómo incluir salvaguardas",
-    "body": "Las salvaguardas son una capa adicional de resguardo para su cuenta en el caso de que pierda la clave de respaldo. Una vez configuradas las salvaguardas, podrá restaurar su cuenta con la ayuda de dos amigos o miembros de su familia. ",
+    "body": "Las salvaguardas son una capa adicional de resguardo para tu cuenta en el caso de que pierda la clave de cuenta. Una vez configuradas las salvaguardas, podrá restaurar tu cuenta con la ayuda de dos amigos o miembros de tu familia. ",
     "warning": "NO DEBE REVELARLE A NADIE LAS IDENTIDADES DE SUS SALVAGUARDAS.",
     "skip": "Omitir por ahora"
   },
   "socialBackupPhraseHeader": "Frase de protección {{index}}",
-  "backupQuizWordCount": "¿Cuál es la palabra <0>{{{ordinal}}</0> de su clave de cuenta?",
+  "backupQuizWordCount": "¿Cuál es la palabra <0>{{{ordinal}}</0> de tu Clave de Cuenta?",
   "socialBackup": {
     "body": "Comparta cada frase que aparece a continuación con un amigo. Asegúrese de enviar solo una frase a cada persona.",
     "confirmation": "He enviado cada una de las salvaguardas a un amigo de confianza.",
-    "yourSafeguards": "Sus salvaguardas"
+    "yourSafeguards": "Tus salvaguardas"
   },
   "cancelDialog": {
-    "title": "Termine de configurar su Clave de Cuenta",
-    "body": "Las claves de la cuenta son extremadamente importantes. Sin una llave, puedes perder el acceso a tu billetera para siempre. Estarás bloqueado para enviar cualquier transacción hasta que termines de configurar tu Clave de Cuenta",
+    "title": "Termine de configurar tu Clave de Cuenta",
+    "body": "Las claves de la cuenta son extremadamente importantes. Sin una clave, puedes perder el acceso a tu app para siempre. Estarás bloqueado para enviar cualquier transacción hasta que termines de configurar tu Clave de Cuenta",
     "action": "Completar ahora",
     "secondary": "Establecer más tarde"
   },
   "backupPhrasePlaceholder": "caballo leopardo perro mono tiburón tigre lémur ballena calamar lobo ardilla ratón león elefante gato langostino oso pingüino ciervo tortuga zorro cebra cabra jirafa",
   "guide": [
     {
-      "title": "La clave de su cuenta es un tipo especial de contraseña",
-      "text": "La clave de su cuenta es la única manera de acceder a su cuenta"
+      "title": "Tu Clave Cuenta es un tipo de contraseña especial",
+      "text": "La Clave Cuenta es la única manera de acceder a tu cuenta"
```

Comment:
```suggestion
      "text": "La Clave de Cuenta es la única manera de acceder a tu cuenta"
```

Reply:
denisse specifically wrote this one out this way, I had a similar question on consistency, so perhaps skip this for now & we can discuss in #spanish?

Reply:
update! this was a typo and has been corrected :)